support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/serializer/JacksonJsonSerializer.java

```diff
@@ -0,0 +1,76 @@
+package org.apereo.cas.serializer;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import org.apereo.cas.TicketSerializer;
+import org.apereo.cas.ticket.ServiceTicket;
+import org.apereo.cas.ticket.ServiceTicketImpl;
+import org.apereo.cas.ticket.Ticket;
+import org.apereo.cas.ticket.TicketGrantingTicket;
+import org.apereo.cas.ticket.TicketGrantingTicketImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+
+import java.io.IOException;
+
+/**
+ * @author David Rodriguez
+ *
+ * @since 5.1.0
+ */
+public class JacksonJsonSerializer implements TicketSerializer<String> {
```

Comment:
Please add 1-2 lines to describe what this component does. It's not immediately obvious, but one could probably guess that tickets are serialized as text and in JSON. It's perhaps useful to describe when one would want to use this vs the binary equivalent.